arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp

```diff
@@ -154,6 +153,40 @@ void IResearchRocksDBRecoveryHelper::PutCF(uint32_t column_family_id,
 
   transaction::StandaloneContext ctx(coll->vocbase());
 
+  // FIXME: check ticks range and possibly omit this step
+  containers::FlatHashSet<IndexId> skipInsert;
+  mustReplay = false;
+  {
+    for (auto const& link : links) {
+      if (link.second) {
+        // link excluded from recovery
+        _skippedIndexes.emplace(link.first->id());
```

Comment:
Looks like unnecessary push

Reply:
If we won't push here  - we will lose this info when we exit on line 187. Imagine we have 2 links. One is excluded from recovery and the second skips insert as document exists in the committed snapshot. In this case we exit without starting inserts loop.

Reply:
@Dronplane